tensorflow/python/compiler/tensorrt/test/tf_trt_integration_test_base.py

```diff
@@ -315,7 +315,10 @@ def _RunGraph(self,
             for expected_shape, actual_val in zip(expected_shapes, new_val):
               self.assertEqual(list(expected_shape), list(actual_val.shape))
             if val is not None:
-              self.assertAllClose(val, new_val, atol=1.e-06, rtol=1.e-06)
+              self.assertAllClose(
```

Comment:
As chatted offline, please revert the changes here or just apply it to specific test (if applicable) instead.
Thanks.

Reply:
@aaroey I reverted this change.

Reply:
Though a similar change was made recently:

```
              # Some ops may have nondeterministic output. E.g. Conv2D may use
              # winograd algorithm. So we set atol/rtol be larger than 1.e-06.
              self.assertAllClose(val, new_val, atol=1.e-05, rtol=1.e-05)
```